codegen/src/main/java/software/amazon/awssdk/codegen/poet/paginators/PaginatorsClassSpec.java

```diff
@@ -169,7 +169,7 @@ protected MemberModel memberModelForResponseMember(String input) {
     protected CodeBlock hasNextPageMethodBody() {
         if (paginatorDefinition.getMoreResults() != null) {
             return CodeBlock.builder()
-                    .add("return $N.$L.booleanValue()",
```

Comment:
1. Can we please write a codegen test  with expected java class and codegen class?
2. Also in what cases do we get this as Null ? Can we write a test case for the same

Reply:
Added tests
The service will always populate the field, but customer unit tests may not populate the more_results field in the response because it previously did not exist